### ISSUE TITLE
Enabling IPv6 support

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/rnpgp.com#13]
We received a kind report from @tambry on the following:

> I recently happened on the RNP website while I had only IPv6 enabled. Unfortunately it lacks IPv6 support and thus failed to load.
> Since it seems to use CloudFront, fixing this should be as easy as enabling IPv6 on the CloudFront distribution and creating an AAAA ALIAS record pointing to it.
> Might be something you'll want to fix.

This we need to fix in the underlying Terraform module (https://github.com/riboseinc/terraform-aws-s3-cloudfront-website) and updating the site to use the new module with the AAAA records. The S3 bucket in the module needs to also have IPv6 enabled.

**Comment by ronaldtse** [comment]
@phuonghyunh can you help this? Thanks!

**Comment by phuonghuynh** [comment]
> Features Not Available over IPv6
> The following features are not currently supported when accessing an S3 bucket over IPv6:
> Static website hosting from an S3 bucket
> BitTorrent

@ronaldtse S3 Static website not support IPV6 acess for now, https://docs.aws.amazon.com/AmazonS3/latest/dev/ipv6-access.html

PR for Cloudfront IPV6 support submitted [here](https://github.com/riboseinc/terraform-aws-s3-cloudfront-website/pull/24)

Tested using a free online IPV6 test sevice for the sample site `http://mysite.booppi.website`.

**Comment by ronaldtse** [comment]
@phuonghuynh I see. Does this mean the cloudfront ipv6 works with S3 ipv4?

**Comment by phuonghuynh** [comment]
Yes, the [PR](https://github.com/riboseinc/terraform-aws-s3-cloudfront-website/pull/24) added new variable to suport it ("true" is default), git [hash](https://github.com/riboseinc/terraform-aws-s3-cloudfront-website/pull/24/commits/6610a19152841f015c54b31d46f9259ae3c1e87b)
